packages/suite/src/hooks/suite/useMessageSystemFirmwareRevisionCheck.ts

```diff
@@ -0,0 +1,10 @@
+import { Feature, selectIsFeatureDisabled } from '@suite-common/message-system';
```

Comment:
please just use selector directly in component, we try to omit hooks

Reply:
Will do. I took inspiration [in your PR](https://github.com/trezor/trezor-suite/pull/12693/commits/b7435db02aeb760bcab32c02d6ff44e2ada7fabe#diff-3f8c84c46f18b1531c7f9f16a33d7e454454f0683f0b08f2f24bf5e056425c31), but I agree that in this case it is unnecessary, since it only returns _one value_.